src/resolvers.js

```diff
@@ -0,0 +1,38 @@
+import fetch from "node-fetch";
+
+const github_data = {
+  token: "ghp_W.......",
+  userName: "PDhanrajnath",
+  repo: "Apollo-GraphQL-React",
+};
+
+const headersForAuth = {
+  "Content-Type": "application/json",
+  Authorization: "Bearer " + github_data["token"],
+};
+
+const resolvers = {
+  Query: {
+    getUser: async (_parent, context) => {
+      console.log(context.Authorization);
+      const response = await fetch("https://api.github.com/user", {
```

Comment:
Get the base url from constants/env file
it will be easier to replace in different environments

Reply:
ok Rohith